gradle-conjure/build.gradle

```diff
@@ -24,7 +24,8 @@ dependencies {
     compile project(':gradle-conjure-api')
     compile 'com.google.guava:guava'
     compile 'com.palantir.sls.versions:sls-versions'
-    compile 'com.fasterxml.jackson.core:jackson-databind'
+    compile 'com.palantir.conjure.java.runtime:conjure-java-jackson-serialization'
```

Comment:
do we _really_ need all this? jackson afterburner etc?

Reply:
I just wanted access to a consistent well configured object mapper, wasn't overly concerned about all the dependencies